cypress/e2e/tests/migration/reports-tab/reports.test.ts

```diff
@@ -39,7 +39,7 @@ const unknownRiskApps = 2;
 
 let riskType = ["low", "medium", "high", "low", "high", "high"];
 
-describe(["@tier2"], "Reports tests", () => {
+describe(["@tier2"], "MTA-2676: Reports tests", () => {
```

Comment:
Please, add "Bug" at the start

`Bug MTA-2676:`

Reply:
done